modules/sms/src/Sms.ts

```diff
@@ -143,19 +146,30 @@ export default class Sms extends ManagedModule<Config> {
     const smsConfig = ConfigController.getInstance().config;
     const name = smsConfig.providerName;
     const settings = smsConfig[name];
-
-    if (name === 'twilio') {
-      try {
-        this._provider = new TwilioProvider(settings);
-      } catch (e) {
-        this._provider = undefined;
-        ConduitGrpcSdk.Logger.error(e as Error);
-        return;
+    try {
+      switch (name) {
+        case 'twilio':
+          this._provider = new TwilioProvider(settings);
+          break;
+        case 'aws':
```

Comment:
rename to aws-sns and the provider to AwsSnsProvider. AWS contains more than one messaging solutions and just adding "SNS" might confuse the users